docs/source/releases.md

```diff
@@ -5,15 +5,16 @@ We are aiming to ship a new release approximately every 6 weeks. The following r
 
 | Release | Code freeze | General availability |
 | :-----: | :---------: | :------------------: |
-| 1.6     | 2021-10-20  | 2021-11-03           |
+| 1.7     | 2021-01-07  | 2021-01-14           |
```

Comment:
move it to the first following Wednesday :)

Reply:
don't everybody loves relases on Fridays? Changed ;)